### PR TITLE
[DO NOT MERGE] AF-2832: Conscious language initiative appformer

### PR DESCRIPTION
**Pull request by akumar074**
**Thank you for submitting this pull request**

**JIRA**: 
[AF-2832](https://issues.redhat.com/browse/AF-2832)
[AF-2829](https://issues.redhat.com/browse/AF-2829)

**referenced Pull Requests**: 
* https://github.com/kiegroup/kie-soup/pull/219
* https://github.com/kiegroup/kie-wb-playground/pull/117
* https://github.com/kiegroup/optaplanner-wb/pull/401
* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1708
* https://github.com/kiegroup/kie-wb-common/pull/3665
* https://github.com/kiegroup/drools-wb/pull/1509
* https://github.com/kiegroup/jbpm-designer/pull/914
* https://github.com/kiegroup/jbpm-wb/pull/1509
* https://github.com/kiegroup/kie-wb-distributions/pull/1118

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by akumar074** [comment]
Jenkins run cdb

**Review by tiagobento — APPROVED** [review]

**Comment by akumar074** [comment]
Jenkins run cdb

**Comment by akumar074** [comment]
Jenkins run cdb

**Comment by akumar074** [comment]
Jenkins run cdb

File: uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/test/java/org/guvnor/m2repo/backend/server/GuvnorM2RepositoryTest.java [hunk 1]
```diff
@@ -200,7 +200,7 @@ public void testDeployArtifactWithDeployArtifactDistributionManagement() throws
                                             // length of plaintext password, obviously not
                                             // length of encrypted password
                                             assertEquals("Plaintext pw (repopw) length expected.",
-                                                         6,
+                                                         46,
```
[the remark this answers is not in the document]
Comment (akumar074) [reply uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/test/java/org/guvnor/m2repo/backend/server/GuvnorM2RepositoryTest.java:203]: Sorry, This is not related! Will remove it.

[the remark this answers is not in the document]
Comment (akumar074) [reply uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/test/java/org/guvnor/m2repo/backend/server/GuvnorM2RepositoryTest.java:203]: Sorry, This is not related! Will remove it.

File: uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/test/java/org/uberfire/java/nio/fs/jgit/JGitMirrorTest.java [hunk 2]
```diff
@@ -64,12 +64,12 @@ public void testToHTTPMirrorSuccess() throws IOException, GitAPIException {
         assertThat(cloned.getRepository().getAllRefs()).is(new Condition<Map<String, Ref>>() {
             @Override
             public boolean matches(final Map<String, Ref> refs) {
-                final boolean hasMasterRef = refs.get("refs/heads/master") != null;
+                final boolean hasMainRef = refs.get("refs/heads/master") != null;
                 final boolean hasNewWebsiteRef = refs.get("refs/heads/new-website") != null;
-                final boolean hasRemoteOriginMaster = refs.get("refs/remotes/origin/master") != null;
+                final boolean hasRemoteOriginMain = refs.get("refs/remotes/origin/master") != null;
```
[the remark this answers is not in the document]
Comment (akumar074) [reply uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/test/java/org/uberfire/java/nio/fs/jgit/JGitMirrorTest.java:69]: @barboras7 I agree that readability is affected due to changing only the variable name. However, this doesn't impact the user interaction with BC. In this case, actual refs can be changed to main once we have the default branch as `main`. I made similar changes based on the impact of user interaction. I have a suggestion if we can do something like this.
```suggestion
                final boolean hasRemoteOriginDefaultBranch = refs.get("refs/remotes/origin/master") != null;
```
wdyt? @tiagobento

File: uberfire-project/uberfire-project-client/src/test/java/org/guvnor/common/services/project/client/security/ProjectControllerTest.java [hunk 3]
```diff
@@ -628,11 +628,11 @@ public void userCannotSubmitChangeRequestTest() {
     }
 
     @Test
-    public void userCannotSubmitChangeRequestWhenInMasterBranchTest() {
+    public void userCannotSubmitChangeRequestWhenInMainBranchTest() {
```
[the remark this answers is not in the document]
Comment (akumar074) [reply uberfire-project/uberfire-project-client/src/test/java/org/guvnor/common/services/project/client/security/ProjectControllerTest.java:631]: Same response as above.

File: uberfire-structure/uberfire-structure-api/src/main/java/org/guvnor/structure/repositories/Repository.java [hunk 4]
```diff
@@ -71,7 +71,7 @@ void addEnvironmentParameter(final String key,
     Collection<Branch> getBranches();
 
     /**
-     * In the case of Git repository this would be master.
+     * In the case of Git repository this would be main.
```
[the remark this answers is not in the document]
Comment (akumar074) [reply uberfire-structure/uberfire-structure-api/src/main/java/org/guvnor/structure/repositories/Repository.java:74]: I think we can keep master here for now.

File: uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/test/java/org/uberfire/java/nio/fs/jgit/JGitMirrorTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (tiagobento) [reply uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/test/java/org/uberfire/java/nio/fs/jgit/JGitMirrorTest.java:69]: +1 for using "default branch".

**Comment by akumar074** [comment]
Jenkins retest this

**Comment by akumar074** [comment]
Jenkins retest this

**Comment by barboras7** [comment]
The changes look good to me. Thanks @akumar074 for the great work you did here! :partying_face:

**Review by barboras7 — APPROVED** [review]

**Pull request merged**